plugins/catalog/src/components/CatalogTable/CatalogTable.tsx

```diff
@@ -52,9 +52,18 @@ export interface CatalogTableProps {
   columns?: TableColumn<CatalogTableRow>[];
   actions?: TableProps<CatalogTableRow>['actions'];
   tableOptions?: TableProps<CatalogTableRow>['options'];
+  emptyContent?: ReactNode;
   subtitle?: string;
 }
 
+const useStyles = makeStyles(theme => ({
+  empty: {
+    padding: theme.spacing(2),
+    display: 'flex',
+    justifyContent: 'center',
+  },
```

Comment:
I wonder if this styling is perhaps a bit too opinionated, should we leave it to the content instead? Especially since it's not possible to override